redaxo/src/core/assets/standard.js

```diff
@@ -466,6 +466,50 @@ jQuery(function($){
     }
 });
 
+// save & restore sidebar panel toggle status via localstorage
+jQuery(function($){
+    var $sidebar = $('#rex-js-main-sidebar'),
+        sidebar_status,
+        $sections;
+    function store_sidebar_status () {
+        sidebar_status = [];
+        $sections.each(function (k, v) {
+            sidebar_status.push($(v).find('div.collapse').hasClass('in').toString());
+        });
+        localStorage.setItem('sidebar_status', JSON.stringify(sidebar_status));
+    }
+    if ($sidebar.length) {
+        sidebar_status = localStorage.getItem('sidebar_status');
+        $sections = $sidebar.children('section');
+        if (!sidebar_status) {
+            store_sidebar_status();
+        }
+        else {
+            sidebar_status = JSON.parse(sidebar_status);
+            if (sidebar_status.length !== $sections.length) {
```

Comment:
würde hier auch dafür sprechen, dass wenn ich zwei verschiedene sidebars habe, die sich immer überschreiben

Reply:
Wow, ich hatte nichtmal auf dem Schirm, dass das ein fragment ist. Habe noch einen commit gemacht, der das Problem lösen sollte - der ganze Kram greift nur auf der content edit Page.

Reply:
das würde das Problem lösen, dann gehört es aber IMO in structure/content und net mehr in den core.

Würde aber vorher gerne von den anderen eine Meinung haben, ob das nur eine Lösung für structure werden soll oder allgemein gültig.

Ich bin für beides offen.

Reply:
Ich würde es erst mal structure/content-spezifisch machen, und dort ablegen.

Reply:
/redaxo/src/addons/structure/plugins/content/assets/sidebar.js oder wie?

Reply:
Jupp

Reply:
Ich hab es erledigt.